listenbrainz/webserver/views/api_compat_deprecated.py

```diff
@@ -134,6 +138,8 @@ def _to_native_api(data, append_key):
                     'track_name': (str)
                 }
             }
+
+        #TODO(param): add tests for this function
```

Comment:
[LB-208](https://tickets.metabrainz.org/browse/LB-208) - can't right now, have a mid semester to study for, but I'll do this soon.